### ISSUE TITLE
Can't set up sync, `org.freedesktop.DBus.Error.ServiceUnknown`

### ISSUE THREAD

**Issue by melianmiko** [mrvladus/Errands#136]
OS: Arch Linux
App: Errands, io.github.mrvladus.List, 45.1.3, from flathub

When trying to set up sync in settings, got this in log. No matter is Nextcloud or CalDAV selected.
```
[INFO] Sync: Initialize sync provider
[INFO] Sync: Initialize CalDAV sync provider
[DEBUG] Sync: Checking credentials
Traceback (most recent call last):
  File "/app/share/errands/errands/widgets/preferences.py", line 137, in on_test_connection_btn_clicked
    res: bool = Sync.test_connection()
                ^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/errands/errands/utils/sync.py", line 49, in test_connection
    self.init(testing=True, window=self.window)
  File "/app/share/errands/errands/utils/sync.py", line 29, in init
    self.provider = SyncProviderCalDAV("CalDAV", self.window, testing)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/errands/errands/utils/sync.py", line 65, in __init__
    if not self._check_credentials():
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/errands/errands/utils/sync.py", line 76, in _check_credentials
    self.password: str = GSettings.get_secret(self.name)
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/errands/errands/utils/gsettings.py", line 45, in get_secret
    return Secret.password_lookup_sync(SECRETS_SCHEMA, {"account": account}, None)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gi.repository.GLib.GError: g-dbus-error-quark: org.freedesktop.DBus.Error.ServiceUnknown (2)
Traceback (most recent call last):
  File "/app/share/errands/errands/widgets/preferences.py", line 129, in on_sync_provider_selected
    self.setup_sync()
  File "/app/share/errands/errands/widgets/preferences.py", line 124, in setup_sync
    password = GSettings.get_secret(account)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/errands/errands/utils/gsettings.py", line 45, in get_secret
    return Secret.password_lookup_sync(SECRETS_SCHEMA, {"account": account}, None)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gi.repository.GLib.GError: g-dbus-error-quark: org.freedesktop.DBus.Error.ServiceUnknown (2)
```

Maybe your app require some additional dependencies outside flatpak?

**Comment by melianmiko** [comment]
Installed version from AUR (aka build from sources), it works fine.

**Comment by mrvladus** [comment]
Looks like flatpak issue, try to upgrade system, run `flatpak update`, and reboot

**Comment by melianmiko** [comment]
```
~ hp-laptop $ flatpak update
Looking for updates…

Nothing to do.
~ hp-laptop $ flatpak run io.github.mrvladus.List 
[DEBUG] Starting Errands 45.1.3
[DEBUG] Initialize GSettings
[DEBUG] Creating actions
[INFO] Data: Clean deleted
[DEBUG] Lists: Update UI...
Traceback (most recent call last):
  File "/app/share/errands/errands/widgets/preferences.py", line 129, in on_sync_provider_selected
    self.setup_sync()
  File "/app/share/errands/errands/widgets/preferences.py", line 124, in setup_sync
    password = GSettings.get_secret(account)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/share/errands/errands/utils/gsettings.py", line 45, in get_secret
    return Secret.password_lookup_sync(SECRETS_SCHEMA, {"account": account}, None)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gi.repository.GLib.GError: g-dbus-error-quark: org.freedesktop.DBus.Error.ServiceUnknown (2)
```
After reboot got the same error.

**Comment by mrvladus** [comment]
Try this: `flatpak run --talk-name=org.freedesktop.secrets io.github.mrvladus.List`

**Comment by melianmiko** [comment]
>  Try this: `flatpak run --talk-name=org.freedesktop.secrets io.github.mrvladus.List`

Yep, now it works as expected.
```
~ hp-laptop $ flatpak run --talk-name=org.freedesktop.secrets io.github.mrvladus.List
[DEBUG] Starting Errands 45.1.3
[DEBUG] Initialize GSettings
[DEBUG] Creating actions
[INFO] Sync: Initialize sync provider
[INFO] Sync: Initialize Nextcloud sync provider
[DEBUG] Sync: Checking credentials
[DEBUG] Sync: Checking URL
[DEBUG] Sync: URL is set to https://nextcloud.vpn/remote.php/dav/
[DEBUG] Sync: Attempting connection
[INFO] Sync: Connected to Nextcloud server at 'https://nextcloud.vpn/remote.php/dav/'
```

**Comment by mrvladus** [comment]
Ok, thanks, I'll add this to flatpak permissions

**Comment by melianmiko** [comment]
Fine, waiting for a new release. For now I'll use version built from sources.

**Comment by mrvladus** [comment]
Fixed in [45.1.4](https://github.com/mrvladus/Errands/releases/tag/45.1.4)
